### ISSUE TITLE
OpenResearch Case Study Title & Description

### ISSUE THREAD

**Issue by hulsiejames** [udsleeds/openinfra#135]
Following on from mine and Greta's interview with Dorka, an Open Research Case Study has been made: https://sway.office.com/NnqQN9uI4SJ4BuXf?ref=Link 

Currently Dorka requires a project title and description from us - I have suggested we use the current GitHub title: 

"Open access data for transport research: tools, modelling and simulation" 

As for the project description (max 1 paragraph), I think the first couple of sentences for turing mobility abstract summarises the project quite efficiently:  
" 
Investment in active travel modes contributes to tackling climate, health, and cost-of-living crises. New policies, investment programs, and government agencies demand high quality, geographically specific data to ensure effective resource allocation. 

The openInfra project explores the potential of open data sources to meet this demand. Specifically, OpenStreetMap (OSM), a free, open, crowdsourced map of the entire planet, is explored. We hypothesise that OSM data can support local decision-making processes, and the prioritisation of new active travel infrastructure. 
" 

Any thoughts/suggestions to the project description are welcome! @GretaTimaite @Robinlovelace

**Comment by GretaTimaite** [comment]
Thanks for creating an issue on this. However, I'm not sure the link to the case study should be here as it was, I think, sent to us for personal use (I don't think it's public yet as she sent it to us to get our approval). If you got a permission then no probs from me!

This sounds like a good para (two paras?) to send. I reckon we can simply combine both paragraphs into one:

> Investment in active travel modes contributes to tackling climate, health, and cost-of-living crises. New policies, investment programs, and government agencies demand high quality, geographically specific data to ensure effective resource allocation. The **OpenInfra** project **explores** the potential of open data sources to meet this demand. Specifically, OpenStreetMap (OSM), a free, open, crowdsourced map of the entire planet, is explored. We hypothesise that OSM data can support local decision-making processes, and the prioritisation of new active travel infrastructure.

Note: I highlighted _OpenInfra explores_ to guide the reader to the aim of the project. 

Question: In the quatation that James included we have openinfra written in lower case but it should be OpenInfra, I believe? Is it a typo in the abstract or in your citation of it here @hulsiejames?

**Comment by hulsiejames** [comment]
Thanks for the comments, Greta!  

I should have mentioned I planned on combining it into a single paragraph, I had formatted it like that so it was easier for myself to read as I typed it up.  

Yes, I had missed the O from openinfra when writing up this issue the other night, good spot. I'll send this over to Dorka now so that she can get everything wrapped up before the end of the week. Thank you for your assistance!
